lms/envs/common.py

```diff
@@ -2088,3 +2088,19 @@
         'profile_image',
     ],
 }
+
+# PROFILE IMAGE CONFIG
+# TODO: add these settings to aws.py as well
```

Comment:
Please have someone from devops review this configuration setting so we know where it will need to live. If we need to add things to aws.py, let's do it now.